docs/Datagrid.md

```diff
@@ -832,6 +832,15 @@ const PostList = () => (
 
 **Tip**: The `Datagrid` component `classes` can also be customized for all instances of the component with its global css name `"RaDatagrid"` as [describe here](https://marmelab.com/blog/2019/12/18/react-admin-3-1.html#theme-overrides)
 
+## Headers pinning
+
+`<Datagrid>` pins headers by default and you don't need to do anything extra for it to work:
```

Comment:
```suggestion
`<Datagrid>` has sticky headers, which means that the header row will remain visible even when scrolling down the page.
```

Reply:
Here I'm using several names for the same feature (sticky headers, header pinning) to make it more discoverable